ivy/functional/frontends/paddle/tensor/random.py

```diff
@@ -99,11 +99,21 @@ def standard_normal(shape, dtype=None, name=None):
 )
 @to_ivy_arrays_and_back
 def randint_like(x, low=0, high=None, dtype=None, name=None):
-    if high == None:
+    if high is None:
         high = low
         low = 0
         if high <= 0:
             raise ivy.exceptions.IvyError(
                 "If high is None, low must be greater than 0, but received low = 0."
             )
-    return ivy.randint(low, high, shape=x.shape, dtype=dtype, seed=None)
\ No newline at end of file
+    return ivy.randint(low, high, shape=x.shape, dtype=dtype, seed=None)
+
+
+@with_supported_dtypes(
+    {"2.5.0 and below": ("float32", "float64")},
+    "paddle",
+)
+@to_ivy_arrays_and_back
+def multinomial(x, num_samples=1, replacement=False, name=None):
+    n = ivy.astype(ivy.array(x.size), ivy.int64)
```

Comment:
No need for doing this since we already have the probabilities in `x` and we are passing them to the `probs` parameter in the `ivy.multinomial` function. We can just equate `n` to `num_samples + 1` since it won't matter anyways because we are passing in the `probs` explicitly but we still need to pass this in as a required positional argument and so it a valid value for this would be `num_samples + 1`.